packages/mds-transactions-api/handlers/get-transactions.ts

```diff
@@ -14,28 +14,154 @@
  * limitations under the License.
  */
 
-import { TransactionServiceClient, TransactionDomainModel } from '@mds-core/mds-transactions-service'
+import {
+  TransactionServiceClient,
+  TransactionDomainModel,
+  PaginationLinks,
+  SORTABLE_COLUMNS,
+  SORTABLE_COLUMN,
+  SORT_DIRECTIONS,
+  SORT_DIRECTION,
+  TransactionSearchParams
+} from '@mds-core/mds-transactions-service'
 import { ApiRequestParams } from '@mds-core/mds-api-server'
 import { parseRequest } from '@mds-core/mds-api-helpers'
-import { ServerError } from '@mds-core/mds-utils'
+import { ServerError, ValidationError } from '@mds-core/mds-utils'
 import { TransactionApiRequest, TransactionApiResponse } from '../@types'
 
 export type TransactionApiGetTransactionsRequest = TransactionApiRequest &
   ApiRequestParams<'provider_id' | 'start_timestamp' | 'end_timestamp'>
 
-export type TransactionApiGetTransactionsResponse = TransactionApiResponse<{ transactions: TransactionDomainModel[] }>
+export type TransactionApiGetTransactionsResponse = TransactionApiResponse<{
+  transactions: TransactionDomainModel[]
+  links: PaginationLinks
+}>
+
+const getOrderOption = (req: TransactionApiGetTransactionsRequest) => {
+  const { order_column: column } = parseRequest(req)
+    .single({
+      parser: x => {
```

Comment:
is there a more descriptive var other than `x`? Maybe `option_key`?

Reply:
Totes